common/achievement/template.ts

```diff
@@ -57,6 +58,17 @@ function buildMetricCache(templates: achievement_template[]) {
     }
 }
 
+function buildRelationCache(templates: achievement_template[]) {
+    achievementTemplates.set(TemplateSelectEnum.BY_RELATION, new Map());
+    achievementTemplates[TemplateSelectEnum.BY_RELATION] = new Map();
```

Comment:
What is the purpose of this line?

Reply:
Yeah, that was a mistake I suppose.

<img src="https://media1.giphy.com/media/N8QDhPtesbzLuBhR4x/giphy.gif"/>

Reply:
😅